assets/src/app/order/nlp_viewer.component.less

```diff
@@ -0,0 +1,27 @@
+@import '../colors';
+
+.nlp {
+	margin-bottom: 100px;
+}
```

Comment:
fit into sub-classes:
```less
.nlp {
	margin-bottom: 100px;
        .ner {
        	background-color: #ABD1FF;
	        margin: 5px;
        	border-radius: 5px 5px 5px;	
        }
}
```
it will generate `.nlp .ner` selector automatically

Reply:
You can repeat for all classes in this file ...